### PR TITLE
fix: set java twitter image

### PR DESCRIPTION
**Pull request by jacobparis**
## Description

Fixes open graph images not working on non gitpod.io domains
Adds the social image to the Java Webinar page

## Related Issue(s)

Fixes #

## Release Notes

```release-note
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2800"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2800"></a>

### CODE REVIEW

**Comment by jacobparis** [comment]
Alright it's all good now


image

**Review by Siddhant-K-code — APPROVED** [review]

File: src/lib/components/open-graph.svelte [hunk 1]
```diff
@@ -14,7 +14,9 @@
     norobots = false,
   } = data || {};
 
-  const url = $page.url.toString();
+  const url = $page.url;
```
Comment (Siddhant-K-code) [inline src/lib/components/open-graph.svelte:17]: why did you change it? I think we can pass `url` on Line32 & Line36
